scripts/normalize.py

```diff
@@ -40,10 +40,10 @@ def normalization(anno_file, norm_outfile, norm_method):
         features=normalize_config["features"],
         image_features=normalize_config["image_features"],
         # meta_features=normalize_config["meta_features"],
-        meta_features=meta_features, # this is 
+        meta_features=meta_features,
```

Comment:
I think this line is a bit wonky. Don't you mean to delete and uncomment line 42?

Reply:
commentes removed: https://github.com/WayScience/CytoPipe/pull/5/commits/d528cad02d082cf0423c424e46a305e6e8a25931